### PR TITLE
List back dev

### PR DESCRIPTION
**Pull request by BlueSkyAndSomeCurses**
added Task reference to task object returned in useFetchTaskList
Chaged [TasksDemo.tsx](https://github.com/FluxonApps/ucu-day-planner-app/compare/list-back-dev?expand=1#diff-55d9f85ae069669e1c284f39305ea0e99e023340d800dbe54a498c8398335d3e) to work properly with my function.

### CODE REVIEW

**Comment by BlueSkyAndSomeCurses** [comment]
I have also changed Task model

File: src/components/TaskList.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { HStack, Box, Img, Link, Stack, Text } from '@chakra-ui/react';
+// import { TasksDemo } from '../pages/TasksDemo';
```
Comment (dannymatkovsky) [inline src/components/TaskList.tsx]: If it's commented but still needed add a comment why. If it's not needed anymore then just remove it

File: src/models/Task.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
 export interface Task {
   name: string;
   description: string;
   importance: number;
   deadline: Timestamp;
-  active: boolean;
+  archived: boolean;
   status: boolean;
   uuid: string;
```
Comment (dannymatkovsky) [inline src/models/Task.ts:11]: BTW. As I understand it is a user's id. If so it's better to use `userId` or `user_id` naming as `uuid` is a name of a widely used [library](https://www.npmjs.com/package/uuid) in js world, so `uuid` is ambiguous.

File: src/models/Task.ts [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+  ref: DocumentReference;
```
Comment (dannymatkovsky) [inline src/models/Task.ts:12]: Hmmm.. I wonder what's the reason for this property

File: src/models/Task.ts (hunk 2 above)
Comment (BlueSkyAndSomeCurses) [reply src/models/Task.ts:11]: ok, will fix everything now

File: src/models/Task.ts (hunk 3 above)
Comment (BlueSkyAndSomeCurses) [reply src/models/Task.ts:12]: that one is for updating task's info in the database

**Pull request closed without merging**